app/models/member.rb

```diff
@@ -25,12 +25,14 @@ class Member < ActiveRecord::Base
   has_one :braintree_customer,     class_name: 'Payment::Braintree::Customer'
   has_one :authentication, class_name: MemberAuthentication, dependent: :destroy
   has_many :payment_methods, through: :customer
+  has_many :actions
   has_paper_trail on: [:update, :destroy]
 
   delegate :authenticate, to: :authentication, allow_nil: true
 
-  validates :email, uniqueness: true, allow_nil: true
-  before_save { email.try(:downcase!) }
+  validates :email, uniqueness: { case_sensitive: true }, allow_nil: true
```

Comment:
Vince, it was changed. See https://github.com/SumOfUs/Champaign/pull/734#discussion-diff-85727221R33